docs/components/movement-sensor/gps/gps-rtk.md

```diff
@@ -0,0 +1,149 @@
+          "ntrip_baud": <>,
+          "ntrip_password": <password for NTRIP server>,
+          "ntrip_path": <>,
+          "ntrip_username": <username for NTRIP server>
+        }
+      },
+      "depends_on": [],
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example: USB/Serial" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    {
+      "name": "my_GPS",
+      "type": "movement_sensor",
+      "model": "gps-rtk",
+      "attributes": {
+        "connection_type": "serial",
+        "correction_source": "ntrip",
+        "serial_attributes": {
+          "serial_baud_rate": 115200,
+          "serial_path": "/dev/serial/by-path/<device_ID>"
+        },
+        "ntrip_attributes": {
+          "ntrip_addr": "<ntrip_address>",
+          "ntrip_baud": 38400,
+          "ntrip_password": "<password>",
+          "ntrip_username": "<username>"
+        }
+      },
+      "depends_on": []
+    }
+  ]
+}
+```
+
+Note that the example `"serial_path"` filepath is specific to serial devices connected to Linux systems.
+
+{{% /tab %}}
+{{% tab name="JSON Example: I2C" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    {
+      "name": "my_GPS",
+      "type": "movement_sensor",
+      "model": "gps-rtk",
+      "attributes": {
+        "board": "board",
+        "connection_type": "I2C",
+        "correction_source": "ntrip",
+        "i2c_attributes": {
+          "i2c_baud_rate": 115200,
+          "i2c_addr": 111,
+          "I2c_bus": "<name_of_bus_on_board>",
+        },
+        "ntrip_attributes": {
+          "ntrip_addr": "<ntrip_address>",
+          "ntrip_baud": 38400,
+          "ntrip_password": "<password>",
+          "ntrip_username": "<username>"
+        }
+      },
+      "depends_on": []
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+Name | Inclusion | Type | Default Value | Description
+---- | --------- | ---- | ------------- | ----------
+`board` | depends on connection type | string | - | Required for NMEA over I<sup>2</sup>C; the board connected to the chip. Not required for serial communication.
+`connection_type` | **Required** | string | - | `"I2C"` or `"serial"`, respectively. See [connection configuration info](../connection/).
+`ntrip_addr` | **Required** | string | - | The URL of the NTRIP server from which you get correction data. Connects to a base station (maintained by a third party) for RTK corrections
+`ntrip_username` | **Required** | string | - | Username for the NTRIP server
+`ntrip_password` | **Required** | string | - | Password for the NTRIP server
+`ntrip_baud` | Optional | int | defaults to `serial_baud_rate`  | Only necessary if you want NTRIP baud rate to be different from serial baud rate.
+`ntrip_connect_attempts` | Optional | int | 10 | How many times to attempt connection before timing out
+`ntrip_mountpoint` | Optional | string | |
+`ntrip_path` | Optional | string | |

```

Comment:
if you know about an rtk mountpoint near you, write its code in here. it will be added to the string making up the ntrip address (ie nysnet.gov/rtcm/NJMTPT1 or something like that) and use that specific mountpoint's data for corrections.

ntrip_path: used for the case where extra hardware is piping rtcm data through a second USB port on a single board computer (like the pi) instead of getting it directly from the internet. It's required now, but might not be in the future (this package is experimental).

Reply:
Sorry, to be clear, are you saying ntrip_path is required now in that case? Or in all current cases?

Reply:
right now - required, but if it can't find the serial path it defaults to a serial path that is directly connected to the GPS chip. I may remove this requirement once we refactor gps-rtk. Yay experimental packages.